packages/core/src/components/ModalNew/ModalHeader/ModalHeader.tsx

```diff
@@ -54,7 +55,7 @@ const ModalHeader = forwardRef(
               />
             )}
             {typeof description === "string" ? (
-              <Text element="span" type={Text.types.TEXT1}>
+              <Text element="span" align={Text.align.INHERIT} type={Text.types.TEXT1}>
```

Comment:
Use strings? Not critical

Reply:
yes, will do, another tech debt from moving the modal from v2 to v3

Reply:
thanks, nice catch